templates/cassiopeia/scss/vendor/bootstrap/_buttons.scss

```
@@ -24,3 +24,8 @@
     background-color: $gray-100;
   }
 }
+@media (max-width: 22.75rem) {  
```

```suggestion
@media (max-width: 22.75rem) {
```
Fix code style error.